.github/workflows/tests.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
           SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
```

Comment:
Why do we have Sauce env vars in the test-local job?

Reply:
Oops, removed, just forgot to remove that when I first split apart the jobs.